content/design-systems-for-developers/react/ko/architecture.md

```diff
@@ -2,60 +2,59 @@
+우리도 바로 이 난관에 봉착했습니다. 경험이 풍부한 팀이 최선의 결과를 위해 노력하는 것이 분명한데도, UI 컴포넌트를 새로 만들거나 이전 것을 가져다 붙여놓는 일이 끊임없이 반복되었습니다. 동일할 거라고 기대한 UI 패턴들이 외관이나 기능에 차이를 보였습니다. 각각의 컴포넌트는 유일무이한 눈 결정처럼 고유해서 새로운 개발자가 기여를 보태기는 커녕 단일한 진실의 원천(source of truth)이 무엇인지조차 파악할 수 없었습니다.
 
-![UIs diverge](/design-systems-for-developers/design-system-inconsistent-buttons.jpg)
+![UIs diverge(분기)](/design-systems-for-developers/design-system-inconsistent-buttons.jpg)
 
 ## 디자인 시스템 만들기
 
-디자인 시스템은 패키지 매니저를 통해 배포된 저장소에서 공통적으로 사용하는 UI 컴포넌트들을 통합시킵니다. 이로써 개발자는 여러 프로젝트에 동일한 UI 코드를 붙여넣지 않고, 표준회된 UI 컴포넌트를 사용하게 됩니다.
+디자인 시스템은 패키지 매니저를 통해 배포되는 중앙 저장소에서 공통으로 사용되는 UI 컴포넌트를 통합시킵니다. 이로써 개발자는 여러 프로젝트에 동일한 UI 코드를 붙여넣지 않고 표준화된 UI 컴포넌트를 불러들여 사용할 수 있습니다.
 
-대부분의 디자인 시스템은 처음부터 만들어진 게 아닙니다. 실제 회사에서 사용 중인 검증된 UI 컴포넌트들이 디자인 시스템으로 재조립된 것입니다. 우리 프로젝트도 예외는 아닙니다. 시간을 절약하고,이해 관계자들에게 보다 신속하게 디자인 시스템을 제공하기 위해 기존 프로덕션 컴포넌트 라이브러리에서 컴포넌트를 선별합니다.
+대부분의 디자인 시스템은 기초부터 새로 만들어지지 않았습니다. 실제 회사에서 사용 중인 검증된 UI 컴포넌트들이 디자인 시스템으로 재조립된 것입니다. 우리 프로젝트도 예외는 아닙니다. 시간을 절약하고 이해 관계자들에게 보다 신속하게 디자인 시스템을 제공하기 위해 기존 프로덕션 컴포넌트 라이브러리에서 컴포넌트를 선별합니다.
 
-![What's in a design system](/design-systems-for-developers/design-system-contents.jpg)
+![디자인 시스템안에 있는 것들](/design-systems-for-developers/design-system-contents.jpg)
 
 ## 디자인 시스템 찾아내기
 
-디자인 시스템을 또 다른 컴포넌트 라이브러리라고 생각할 수 있지만, 하나의 앱을 서비스하는 것이 아니라 전체 조직에 서비스를 제공하는 것입니다. 디자인 시스템은 UI 기본 요소에 초점을 맞추는 반면, 프로젝트별 컴포넌트 라이브러리는 복합적인 컴포넌트부터 화면에 이르기까지 모든 것을 포함할 수 있습니다.
+또 하나의 컴포넌트 라이브러리라고 생각할 수도 있는 디자인 시스템은 사실 특정 앱을 위해서만 활용되는 것이 아니라 전체 조직을 위한 것입니다. 디자인 시스템은 UI 기본 요소에 초점을 맞추는 반면, 특정 프로젝트를 위한 컴포넌트 라이브러리는 복합체 컴포넌트부터 화면에 이르기까지 모든 것을 포함할 수 있습니다.
 
-따라서 디자인 시스템은 모든 프로젝트로부터 독립적이어야 하며, 또한 모든 프로젝트에 종속성이 있어야 합니다. 변경 사항은 패키지 매니저를 통해 배포된 버전으로 조직 전체에 적용됩니다. 이렇게 프로젝트는 디자인 시스템의 컴포넌트를 재사용하고, 필요한 경우에는 추가로 사용자 정의를 할 수 있습니다. 이러한 조건들은 프론트엔드 프로젝트를 개발하는 데에 청사진이 되기도 합니다.
+따라서 디자인 시스템은 모든 프로젝트로부터 독립적이어야 하는 동시에 모든 프로젝트에 종속성이 있어야 합니다. 변경 사항은 패키지 매니저를 통해 배포된 버전으로 조직 전체에 적용됩니다. 이렇게 프로젝트는 디자인 시스템의 컴포넌트를 재사용하고, 필요한 경우에는 추가로 사용자 정의를 할 수 있습니다. 이러한 조건들은 프런트엔드 프로젝트를 개발하는 데에 청사진이 되기도 합니다.
 
-![Who uses a design system](/design-systems-for-developers/design-system-consumers.jpg)
+![디자인 시스템을 사용하는 사람들](/design-systems-for-developers/design-system-consumers.jpg)
 
-## 깃허브와 create-react-app으로 저장소 만들기
+## 깃허브(Github)와 create-react-app으로 저장소 만들기
 
-[State of JS](https://stateofjs.com/)의 설문조사에 따르면, 가장 인기 있는 뷰 레이어는 리액트(React)입니다. 압도적으로 많은 스토리북에서 리액트를 사용하기 때문에, 이 튜토리얼에서도 [create-react-app](https://github.com/facebook/create-react-app) 보일러플레이트를 사용하겠습니다.
+[JS의 State(상태)](https://stateofjs.com/)의 설문조사에 따르면 가장 인기 있는 뷰 레이어는 리액트(React)입니다. 압도적으로 많은 스토리북에서 리액트를 사용하기 때문에 이 튜토리얼에서도 [create-react-app](https://github.com/facebook/create-react-app) 보일러플레이트(Boilerplate)와 함께 리액트를 사용하겠습니다.
 
 터미널에 아래 명령어를 입력하여 실행해보세요.
 
 ```shell
-# Clone the files
+# 파일들을 복제합니다.
 npx degit chromaui/learnstorybook-design-system#setup learnstorybook-design-system
 
 cd learnstorybook-design-system
 
-# Install the dependencies
+# 의존성(dependencies)를 설치합니다.
 yarn install
 ```
```

Comment:
@lisohh if you could update this code snippet to match what's used currently in the English version documented [here](https://storybook.js.org/tutorials/design-systems-for-developers/react/en/architecture/) I'd appreciate it. To give you a bit more context, we're no longer using a branch, but instead a dedicated template. Also if you could revert the code comments back to English as I've mentioned previously it would also help out. So instead of:

```shell
# 파일들을 복제합니다.
npx degit chromaui/learnstorybook-design-system#setup learnstorybook-design-system
cd learnstorybook-design-system
# 의존성(dependencies)를 설치합니다.
yarn install
```

Instead it would be:

```shell
# Clone the files
npx degit chromaui/learnstorybook-design-system-template learnstorybook-design-system

cd learnstorybook-design-system

# Install the dependencies
yarn install
```